main/lsp/requests/hover.cc

```diff
@@ -21,14 +22,27 @@ string methodSignatureString(const core::GlobalState &gs, const core::TypePtr &r
         }
         start = start->secondary.get();
     }
+
     return contents;
 }
 
-unique_ptr<MarkupContent> formatRubyCode(MarkupKind markupKind, string str) {
-    if (markupKind == MarkupKind::Markdown && str.length() > 0) {
-        str = fmt::format("```ruby\n{}\n```", str);
+unique_ptr<MarkupContent> formatRubyCode(MarkupKind markupKind, string_view sigString,
```

Comment:
Nit: `sigString` is a bit of a misnomer. It just contains a type, which might not be a `sig`. Can you rename this to `typeString`?

Reply:
Nit: `formatRubyCode` is now a misnomer. Let's rename it to `formatHoverText`.

Reply:
Ah yes, forgot about this! thank you
90c3c22c2
91a1fd893